### DISCUSSION TITLE
Clarifications in README for UpdatesAtTime and ListPath

### DISCUSSION THREAD

**Discussion by flowsworld** [Romanitho/Winget-AutoUpdate#665] [General]
Two small clarifications in the README would be much appreciated...

1. it should be specified what time format UpdatesAtTime requires 
2. the file defined in ListPath can't be empty (at least when hosted on GitHub) otherwise the updater will fail.
```
13:11:48 - WAU uses External Lists from: https://raw.githubusercontent.com/[...]/BlockList
13:11:49 - Couldn't reach/find/compare/copy from https://raw.githubusercontent.com/[...]/BlockList...
13:11:49 - Critical: White/Black List doesn't exist, exiting...
```

**Comment by Romanitho** [comment]
Hi,

Thank you for your feedback.
Feel free to make any changes or additions to the README as you see fit. Your contributions will be included, and you’ll be credited.

**Reply by flowsworld** [reply]
The problem is, I don't know the time format, hence why I would ask for it to be specified.
A notice for the ListPath I could add, but if someone already adds the time format, that could be easily added in one go.

**Reply by KnifMelti** [reply]
That's odd.
The download of the URL list is constructed as:
`$ExternalList = -join ($ListPath, "/", $ListType)`
https://github.com/Romanitho/Winget-AutoUpdate/blob/main/Sources/WAU/Winget-AutoUpdate/functions/Test-ListPath.ps1

That means that if you're using an **excluded_apps.txt** and as **ListPath** have included the file name, then it will be:
`https://raw.githubusercontent.com/[...]/BlockList/excluded_apps.txt/excluded_apps.txt` (and would not work) 

Are you sure the filename in the path is `excluded_apps.txt` and not anything else? 

EDIT: Tested against https://github.com/Romanitho/Winget-AutoUpdate/raw/main/Sources/WAU where an `excluded_apps.txt` is located, found it fine.
Added `excluded_apps.txt` in the Path, failed.

**Reply by KnifMelti** [reply]
Time format is anything, for example 06am or military 06:00

**Reply by flowsworld** [reply]
That's not what I said. I did not include the filename in the ListPath URL. I said, that it will fail, if the file excluded_apps.txt is empty.

**Reply by KnifMelti** [reply]
Ah, that's correct (no point without content, then you shouldn't use an external list)

**Reply by KnifMelti** [reply]
But, thinking about it a little more...
...a situation where you don't want to exclude/include anything needs an empty list.
Maybe we should let an empty list be downloaded from GitHub (it's written the way it is as a check because GitHub doesn't supply any date/time in the header like a standard web server)?

**Reply by KnifMelti** [reply]
#667

**Reply by KnifMelti** [reply]
#673 now takes care of empty lists from GitHub (always downloading)
